### PR TITLE
[RUN-4198] Remove irrelevant parts in the "Debug Java Actions Remotely" document

### PR DESCRIPTION
**Pull request by atennapel**

### CODE REVIEW

File: content/en/docs/howto/monitoring-troubleshooting/debug-java-actions/debug-java-actions-remotely.md [hunk 1]
```diff
@@ -36,12 +34,6 @@ To edit the server configuration with extra JVM parameters, follow these steps:
 
 In your Java IDE, you need to configure remote debugging.
 
-To configure IntelliJ, follow these steps:
-
-1. Start the **Debugger**.
-2. Place some breakpoints in your Java code.
-3. Start debugging.
-
 {{< figure src="/attachments/howto/monitoring-troubleshooting/debug-java-actions/debug-java-actions-remotely/intellij_rundebug_configurations.png" class="no-border" >}}
```
Comment (MarkvanMents) [inline content/en/docs/howto/monitoring-troubleshooting/debug-java-actions/debug-java-actions-remotely.md]: Do we still need this image, it doesn't seem to have the **Debug** button on it and I can't really match it up to the Eclipse configuration - although I can see a reference to _5005_?
intellij_rundebug_configurations

**Review by MarkvanMents — COMMENTED** [review]
@atennapel 
Thanks for the PR, Albert. I just have one question about the image which is still in the doc.
Otherwise, looks ready to go.

File: content/en/docs/howto/monitoring-troubleshooting/debug-java-actions/debug-java-actions-remotely.md (hunk 1 above)
Comment (atennapel) [reply content/en/docs/howto/monitoring-troubleshooting/debug-java-actions/debug-java-actions-remotely.md]: I have updated the image.

**Review by MarkvanMents — APPROVED** [review]
Thanks for responding so quickly.
Approving and merging.

**Pull request merged**